jina/peapods/runtimes/head/__init__.py

```diff
@@ -49,6 +49,19 @@ def __init__(
             k8s_namespace=args.k8s_namespace,
         )
 
+        self._endpoint_polling = {}
```

Comment:
maybe we can bring this line up: `self.polling = args.polling if hasattr(args, 'polling') else PollingType.ANY`